README.md

```diff
@@ -99,4 +99,49 @@ $ docker-compose run bintest
 * [mruby-eso-research](https://github.com/hone/mruby-eso-research) - an app for managing crafting research in Elder Scrolls Online. It uses YAML as the data store.
 
 ## mruby-cli Development
-This app is built as a mruby-cli app, so you just need to run: `docker-compose run compile` and find the binaries in the appropriate directories.
+
+### Compile the mruby-cli binaries
+
+This app is built as a `mruby-cli` app, so you just need to run:
+
+```
+docker-compose run compile
+```
+
+and find the binaries in the appropriate directories.
+
+#### Common error
+
+If rather than typing `docker-compose run compile`, you type `rake compile`, you could get on a nix OS the following error:
```

Comment:
@hone I've just pushed a new commit including both suggestions.

The first check tests if it is run inside a Docker container. That could be maybe soften if on a compatible host.

When compiling, the second check tests the presence of cc command for every target.

What do you think?